### DISCUSSION TITLE
Manage header overflow in android devices

### DISCUSSION THREAD

**Discussion by ghost** [adityatelange/hugo-PaperMod#970] [Q&A] [answered]
How can I implement a hamburger menu as it appears in @adityatelange has on his [website](https://adityatelange.in)?
By default, PaperMod gives no such menu while opening a site in phone, but in the website, it appears.

**Comment by adityatelange** [comment, marked as answer]
Yes. My personal website uses personalized PaperMod. PaperMod does not include a hamburger menu

**Reply by ghost** [reply]
Okay then, can I tweak the css and js files and partials to get the same behaviour? I am still a beginner in this, so it would be very helpful if you could kindly show me what changes to make. Thanks in advance.

**Reply by adityatelange** [reply]
Yes you can tweak. Sorry but I can't share it please figure out yourself.

**Reply by sahashirshendu** [reply]
Thanks
